demos/python_demos/asl_recognition_demo/asl_recognition_demo.py

```diff
@@ -0,0 +1,175 @@
+from asl_recognition_demo.person_detector import PersonDetector
+from asl_recognition_demo.person_tracker import PersonTracker
+from asl_recognition_demo.action_recognizer import ActionRecognizer
+
+DETECTOR_OUTPUT_NAME = "12688/Split.0"
+TRACKER_SCORE_THRESHOLD = 0.5
+TRACKER_IOU_THRESHOLD = 0.5
+ACTION_NET_INPUT_FPS = 15
+ACTION_NUM_CLASSES = 100
+ACTION_IMAGE_SCALE = 256
+ACTION_SCORE_THRESHOLD = 0.8
+
+
+def build_argparser():
+    """ Returns argument parser. """
+
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-m_a', '--action_model',
+                      help='Required. Path to an .xml file with a trained asl recognition model.',
+                      required=True, type=str)
+    args.add_argument('-m_d', '--detection_model',
+                      help='Required. Path to an .xml file with a trained person detector model.',
+                      required=True, type=str)
+    args.add_argument('-i', '--input',
+                      help='Required. Path to a video file or a device node of a web-camera.',
+                      required=True, type=str)
+    args.add_argument('-c', '--class_map',
+                      help='Required. Path to a file with ASL classes.',
+                      required=True, type=str)
+    args.add_argument('-d', '--device',
+                      help='Optional. Specify the target device to infer on: CPU, GPU, FPGA, HDDL '
+                           'or MYRIAD. The demo will look for a suitable plugin for device '
+                           'specified (by default, it is CPU).',
+                      default='CPU', type=str)
+    args.add_argument("-l", "--cpu_extension",
+                      help="Optional. Required for CPU custom layers. Absolute path to "
+                           "a shared library with the kernels implementations.", type=str,
+                      default=None)
+    args.add_argument('--no_show', action='store_true',
+                      help='Optional. Do not visualize inference results.')
+
+    return parser
+
+
+def load_class_map(file_path):
+    """ Returns class names map. """
+
+    if file_path is not None and exists(file_path):
+        with open(file_path, 'r') as input_stream:
+            data = json.load(input_stream)
+            if isinstance(data, dict):
+                keys = list(data)
+                assert len(keys) > 0
+
+                if isinstance(keys[0], str):
+                    class_map = {v: k for k, v in data.items()}
+                elif isinstance(keys[0], int):
+                    class_map = data
+                else:
+                    raise ValueError('Unknown format of the class map has been provided.')
+            elif isinstance(data, list):
+                class_map = dict(enumerate(data))
```

Comment:
Why are there so many options here? The example file is a list, why support anything else?

Reply:
Fixed.